components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -117,10 +134,23 @@ public BrandingPreference resolveBranding(String type, String name, String local
             }
         }
 
-        /* Tenant domain will always be carbon.super for SaaS apps (ex. myaccount). Hence need to resolve
+        if (APPLICATION_TYPE.equals(type)) {
+            return resolveApplicationBranding(name, locale, organizationManager, organizationId, currentTenantDomain);
```

Comment:
`organizationManager` is not a matching param for the private method.
Seems you have passed it to use the same instence service and reduce the code.
but for the clean method signature better to remove it and get organizationManager inside the private methods.

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/40/commits/a9322948eef77b783bd87838edceb408a676e7f4